pwndbg/gdblib/heap/jemalloc.py

```diff
@@ -0,0 +1,488 @@
+from __future__ import annotations
+
+import gdb
+
+import pwndbg.gdblib.info
+import pwndbg.gdblib.memory
+import pwndbg.gdblib.typeinfo
+
+RTREE_HEIGHT = 2
+# adapted from jemalloc source 5.3.0
+LG_VADDR = 48
+LG_PAGE = 12
+RTREE_NLIB = LG_PAGE
+# jemalloc/include/jemalloc/internal/jemalloc_internal_types.h
```

Comment:
Please change to a link to github (or other VCS) with specific version this is based from (5.3.0), or even commit